src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -97,23 +121,23 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
                         }
 
                         return this._getStripeJs().createPaymentMethod('card', this._cardElement,
-                            this._mapStripePaymentMethodOptions()
-                        ).then(stripePaymentMethod => {
-                            if (stripePaymentMethod.error || !stripePaymentMethod.paymentMethod.id) {
-                                throw new StandardError(stripePaymentMethod.error && stripePaymentMethod.error.message);
-                            }
+                            this._mapStripePaymentMethodOptions())
```

Comment:
Can you combine this line with the line above so the indentation for this block remains correct? i.e.: 

```ts
return this._getStripeJs().createPaymentMethod('card', this._cardElement, this._mapStripePaymentMethodOptions())
```